README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
@smarnach I posted before your answer appeared - that seem like a good suggestion, if changing the call doesn't have side effects which would add more work; we were just looking at updating the README originally, I wouldn't want this to bubble up and having to postpone the work. In which PR would this be done?

CC @bradenmacdonald FYI btw, since we are discussing your PR too at the end : )